packages/node_modules/@webex/internal-plugin-ediscovery/src/index.js

```diff
@@ -157,6 +177,28 @@ registerInternalPlugin('ediscovery', EDiscovery, {
 
           return Promise.all(object.body.map((item) => ctx.transform('decryptReportContentSummary', {body: item})));
         }
+      },
+      {
+        name: 'decryptReportContentContainer',
+        direction: 'inbound',
+        fn(ctx, object) {
+          if (!object || !object.body || !object.body.containerName || !object.body.participants || !object.body.participants.length) {
```

Comment:
If a container has no participants don't decrypt the `containerName`. Is that the right behaviour? It may not be possible if the actor is included in the participants (which I forget) as I don't think there is ever a container without an actor (though I'm not 100% about that).

Reply:
BTW it looks like a lot of the same checks are done by the transform. Is this check necessary?

Reply:
Yeah, good point, this is probably a relic from when the plugin had to determine the onBehalfOf user itself from the participant list, with the service now providing it we should probably just be checking for that. Will remove the duplicate checks here also, makes more sense to do them in the transform itself.